addon/content/reducer-messages.js

```diff
@@ -75,7 +79,7 @@ async function setupConversationInTab(params, isInTab) {
       // teach Conversation how to handle those.
       msgUrls,
       isThreaded,
-      ++Conversations.counter,
+      ++window.Conversations.counter,
```

Comment:
Is there an advantage to using `window.Conversations` here? The fact it was in the global ESLint definition was a useful hint that it is something we need to fix.

Reply:
There was some weirdness running in the dev frame without the `window.` in front...It would complain on line 371 about `Conversations` being undefined.

https://github.com/siefkenj/thunderbird-conversations/blob/db4c9b356fdc82c00c39055ebcc385ae55dd3d4b/addon/content/reducer-messages.js#L371

I'm not sure why `Conversations` was not loaded into the global scope...It could have something to do with webpack and the way modules work? In any case, adding a `window.` in front fixed it. It will be nice when we can get rid of that global object...

Reply:
Ok, that's fine, lets keep this as you've got it for now.